### ISSUE TITLE
Unit movement destination is inaccurate

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#2384]
Unit movement destination is inaccurate.

Perhaps the movement coords are rounded to the nearest integer, or has a calculation step that accidentally loses floating precision somewhere.

https://github.com/TheSuperHackers/GeneralsGamePatch/assets/4720891/fec06416-53b4-40b6-a975-aad4d11c4399

**Comment by MTKing4** [comment]
Yeah that is such a big deal in micro battles, i always found that units don't follow my orders as well as i want them to, I began to realize this once i moved over to newer, state of the art tech RTS games with superior micro potential and impressive responsiveness with High FPS rate then come back to try zh :3
